tools/distrib/python/make_grpcio_tools.py

```diff
@@ -228,7 +231,19 @@ def _copy_source_tree(source, target):
             shutil.copyfile(source_file, target_file)
 
 
+def _delete_source_tree(target):
+    """Deletes the copied target directory."""
+    target = GRPCIO_TOOLS_ROOT_PREFIX + target
+    target_abs = os.path.join(*target.split("/"))
+    print("Deleting copied folder %s" % (target_abs))
+    shutil.rmtree(target_abs, ignore_errors=True)
+
 def main():
+    parser = argparse.ArgumentParser()
```

Comment:
actually, looks like that since we weren't parsing arguments of the script before (at all), on older version of make_grpcio_tools.py, the option `--cleanup-third-party` would be silently ignored?

Whether that's something we want or not I'm not sure, but I wanted to point this behavior out.

Also, can we simply make the "--cleanup-third-party" behavior the default (and not need a new cmdline flag at all)?